packages/validator/test/cli-validator/mock-files/oas3/err-and-warn.yaml

```diff
@@ -149,8 +149,6 @@ components:
         name:
           type: string
           description: string
-      xml:
```

Comment:
"xml" is not in our allow-list for openapi 3.1, so I removed instances of this from both the oas3 and oas31 flavor of this API because the "expected-output" test is implemented so that the same test code runs on both flavors of the document.